cat/ptx-without-proxy.cat

```diff
@@ -0,0 +1,54 @@
+PTX
```

Comment:
I just checked and there are litmus tests (without having any proxy feature) for which the two models return different results, e.g., `litmus/PTX/CoRW2.litmus `. Because of this it makes sense to keep both cat files (however I would probably call the original one `pxt` and the latest one `ptx-proxy`). Do we have a way to check the litmus tests wrt to the old alloy model (i.e. the one with no proxy) to see what are the expected results wrt that model?

If we are keeping both models, we probably should have unit tests for both. For the original ptx model, you will need to skip the tests having proxy features. Take a look how we do something similar in `AbstractCompilationTest.getCompilationBreakers()`.

You can use something like below for the model not matching the `ptx.cat` name
```
    @Override
    protected Provider<Wmm> getWmmProvider() {
        return Providers.createWmmFromName(() -> "ptx-without-proxy");
    }
```
You will probably need something similar to say from which file read the expected result (you would need `PTX-expected.csv` and `PTX-proxy-expected.csv`)

Reply:
I didn't found a way to run litmus tests on the original ptx, this is the github link of [ASPLOS'19 paper](https://github.com/NVlabs/ptxmemorymodel).

The difference of two models are mainly caused by the `cause-base` relation
- `let cause-base = (po?; sw; po?)+ // PTX6.0`
- `let cause-base = (po?; sw; po?)+ | po // PTX7.5`

So we might have to use the `ptx-proxy` Alloy tool, for original `ptx` tests after updating their alloy to older version.

Reply:
You can probably just change
```
fun cause_base : Op->Op {
  ^po + *po.^((sc + sync).*po)
}
```
by
```
fun cause_base : Op->Op {
  *po.^((sc + sync).*po)
}
```
in the proxy tool and run the tests there

Reply:
Yes